src/fides/api/api/v1/endpoints/privacy_request_endpoints.py

```diff
@@ -557,6 +557,8 @@ def get_request_status(
 ) -> Union[StreamingResponse, AbstractPage[PrivacyRequest]]:
     """Returns PrivacyRequest information. Supports a variety of optional query params.
 
+    TODO: Why is request_id a query param instead of a path param?
```

Comment:
I believe this is the endpoint that populates the privacy requests page, so by default we return a paginated list of privacy requests.  There are several UI filters on that page, one of them allows you to filter directly by request id.